front/lib/development.ts

```diff
@@ -14,3 +14,8 @@ export function isDevelopmentOrDustWorkspace(owner: WorkspaceType) {
     owner.sId === PRODUCTION_DUST_APPS_WORKSPACE_ID
   );
 }
+
+export function isActivatedStructuredDB(owner: WorkspaceType) {
```

Comment:
Can you replace current use of isDevelopmentOrDustWorkspace?

Reply:
In action selection right?

Reply:
> Can you replace current use of isDevelopmentOrDustWorkspace?

I plan to do it in a next PR, it needs to be replaced in a lot of places and it will be easier to review if that's the only change in the PR 🙏🏻